src/render/components/Podman/container/info.vue

```
@@ -11,138 +11,138 @@
+                    ? t('podman.common.yes')
+                    : t('podman.common.no')
                 }}
               </el-tag>
             </el-descriptions-item>
-            <el-descriptions-item :label="I18nT('podman.container.isInfra')">
+            <el-descriptions-item :label="t('podman.container.isInfra')">
               <el-tag :type="containerDetail.IsInfra ? 'warning' : 'info'">
                 {{
-                  containerDetail.IsInfra ? I18nT('podman.common.yes') : I18nT('podman.common.no')
+                  containerDetail.IsInfra ? t('podman.common.yes') : t('podman.common.no')
                 }}
```

ESLint

[error] Replace `⏎··················containerDetail.IsInfra·?·t('podman.common.yes')·:·t('podman.common.no')⏎···············` with `·containerDetail.IsInfra·?·t('podman.common.yes')·:·t('podman.common.no')`

(prettier/prettier)